build.sbt

```
@@ -389,9 +389,9 @@ lazy val docs = project
 
 lazy val cats = project
   .in(file("."))
-  .settings(moduleName := "root")
-  .settings(catsSettings)
-  .settings(noPublishSettings)
+  .settings(moduleName := "root", crossScalaVersions := Nil)
+  .settings(publishSettings) // these settings are needed to release all aggregated modules under this root module
+  .settings(noPublishSettings) // this is to exclue the root module itself from being published. 
```

```suggestion
  .settings(noPublishSettings) // this is to exclue the root module itself from being published.
```

There is a dangling space at the end of this line. I've removed it in this suggestion.